src/test/java/org/csanchez/jenkins/plugins/kubernetes/PodTemplateBuilderTest.java

```diff
@@ -94,23 +102,43 @@ public void testParseLivenessProbe() {
     }
 
     @Test
-    public void testBuildWithoutSlave() throws Exception {
+    public void testBuildWithoutSlave_directConnection() throws Exception {
```

Comment:
Sounds like something that could be written using https://github.com/junit-team/junit4/wiki/parameterized-tests

Reply:
I first implemented `org.junit.runners.Parameterized` but then switched to [`junitparams.JUnitParamsRunner`](https://github.com/Pragmatists/JUnitParams) since `Parameterized` runs **all** test methods with the configured Parameters, not only the ones where it is wanted.